src/components/LetterBox/LetterBox.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+/** style */
+import * as El from './LetterBox.style'
+// To ensure box and line are the same width
+// TODO: Make this contigent on how many letters are in the word to guess
+
+const LetterBox = ({char, key}) => {
```

Comment:
Getting this in the console:
![image](https://user-images.githubusercontent.com/6750696/95408611-b48a8d00-08d4-11eb-93f1-95171d4e881f.png)

Reply:
fixed!